feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/ui/CommonClaims.kt

```diff
@@ -130,37 +79,36 @@ fun PreviewCommonClaims() {
       selectClaim = {},
       commonClaims = listOf(
         SearchableClaim(
-          id = "1",
+          entryPointId = "1",
           displayName = "Broken phone",
           itemType = ItemType("PHONE"),
         ),
         SearchableClaim(
-          id = "4",
+          entryPointId = "4",
```

Comment:
Since the entryPointId is used as a `key` for LazyColumn, I think if it's not unique it will crash, making this preview also not work? Not sure, but might be worth checking and if it does not work adding a unique entryPointId on each of these preview items.